src/main/java/buildcraftAdditions/armour/ItemKineticBackpack.java

```diff
@@ -19,10 +20,12 @@
  * Please check the contents of the license located in
  * http://buildcraftadditions.wordpress.com/wiki/licensing-stuff/
  */
-public class KineticBackpack extends ItemArmor implements IEnergyContainerItem {
+public class ItemKineticBackpack extends ItemArmor implements IEnergyContainerItem {
```

Comment:
mine are pretty minor and just testing so i'll revert mine, doesn't matter much, just reminding you that you should watch out for things like that